api-module-library/sharepoint/config/default.js

```diff
@@ -0,0 +1,30 @@
+
+if (process.env.NODE_ENV !== 'production') {
+    // Load values from .env
+    require('dotenv').config();
+}
+
+const apiModule = {
```

Comment:
@seanspeaks  Not sure if that's best name for that variable, I guess values in here refer to the module itself?

Reply:
@seanspeaks - naming is hard. This excerpt looks like the configuration or definition of the module. I agree it _looks_ like `metadata.` Still, such an open name could bite us back in the future and start containing virtually anything (as metadata is something like "data about the data").

So my best guesses for this block: "definitions" or "details."

Thinking about the import code, `import { definitions } from '../sharepoint/config'`  or `import { details } from '../sharepoint/config'` sounds more "I have some clue about what is coming here" than `import { metadata } or import { meta }.`

Just my 50 cents.